glam/api/views.py

```diff
@@ -98,11 +98,11 @@ def get_firefox_aggregations(request, **kwargs):
     # Whether to pull aggregations by version or build_id.
     if aggregation_level == "version":
         dimensions.append(Q(build_id="*"))
-        counts = _get_firefox_counts(channel, os, versions, by_build=False)
+        #counts = _get_firefox_counts(channel, os, versions, by_build=False)
```

Comment:
GLAM has stopped using separate counts in favor of `total_users`column embedded in the probe's aggregates